server/model/user.js

```diff
@@ -0,0 +1,55 @@
+const mongoose = require("mongoose");
+const USER = mongoose.Schema(
+  {
+    firstName: {
+      type: String,
+      required: [true, "please add a name "],
+    },
+    middlename: {
+      type: String
+    },
+    surname: {
+      type: String,
+      required: [true, "please add a name "],
+    },
+    // passport:{
+    //     type:String,//will be a string,
+    //     required:[true,"please add a photo"]
+    // },
+    role: {
+      type: String,
+      default: "staff",
+      required: [true, "please specify a role"],
+      default:"default"
+    },
+    email: {
+      type: String,
+      unique:true,
+      required: [true, "please specify a role"],
+    },
+  
+    password: {
+      type: String,
+    },
+    dateOfBirth:{
+type:String,
+required:[true,"include a date of birth"]
+    },

```

Comment:
[OPTIONAL] Date of birth could not be a requirement in order to preserve user privacy. Some staff wont be inclined to give out their DoB to a delivery app.

We could consider leaving this out. It is up to you!